modules/uhsg_domain/uhsg_domain.info.yml

```diff
@@ -0,0 +1,6 @@
+name: UHSG Domain
+type: module
+description: Provides domain functionality for other modules.
```

Comment:
"Domain functionality" doesn't explain that much, could we try something like:
> Provides support for creating guide-content for student and teacher domains.

Same thing would go for `hook_help()`.

Reply:
Tuned.